infra/aws/terraform/service-quotas/boskos/main.tf

```diff
@@ -0,0 +1,112 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+terraform {
+  required_version = "~> 1.3.0"
+
+  backend "s3" {
+    bucket = "eks-e2e-boskos-tfstate"
```

Comment:
Where this bucket is located? Maybe we could create it with CF?

Reply:
That's a good idea, but let's leave it for follow up.

Reply:
At least, can we comment about in which account it's created ?

Reply:
That's a good point @ameukam, will be added.

Reply:
S3 bucket is now managed via CloudFormation as well.